src/System.CommandLine/Binding/ArgumentConverter.cs

```diff
@@ -262,7 +263,7 @@ public static bool CanBeBoundFromScalarValue(this Type type)
             }
             else
             {
-                info = (null, null);
+                info = null;
```

Comment:
nit: not sure if it matters, but have you considered returning two `out` parameters, to avoid boxing tuple?